src/instructlab/sdg/pipelines/schema/v1.json

```diff
@@ -0,0 +1,151 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["version", "blocks"],
+  "properties": {
+    "version": {
+      "type": "string"
+    },
+    "blocks": {
+      "type": "array",
+      "additionalProperties": false,
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["name", "type", "config"],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "type": {
+            "type": "string"
+          },
+          "drop_duplicates": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "drop_columns": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "gen_kwargs": {
+            "type": "object",
+            "properties": {
+              "max_tokens": {
+                "type": "number"
+              },
+              "temperature": {
+                "type": "number"
+              },
+              "n": {
+                "type": "number"
+              }
+            }
+          },
+          "config": {
+            "anyOf": [
+              {
+                "type": "object",
+                "description": "FilterByValueBlock",
+                "additionalProperties": true,
+                "properties": {
+                  "config_path": {
+                    "type": "string"
+                  },
+                  "output_cols": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    }
+                  },
+                  "convert_dtype": {
+                    "type": "string"
+                  },
+                  "filter_column": {
+                    "type": "string"
+                  },
+                  "filter_value": {
+                    "oneOf": [
+                      {
+                        "type": "string"
+                      },
+                      {
+                        "type": "number"
+                      }
+                    ]
+                  },
+                  "operation": {
+                    "type": "string"
```

Comment:
You could list the supported ones?

Reply:
`"enum": ["eq", "ne", "gt", "ge", "lt", "le", "contains"]`

but I think you're trying to get rid of `contains`?

Reply:
Fixed